KappaAnalysis/interface/Producers/ValidMuonsProducer.h

```diff
@@ -76,7 +76,8 @@ class ValidMuonsProducer: public ProducerBase<TTypes>, public ValidPhysicsObject
 		else if (muonID == "fakeable") return MuonID::FAKEABLE;
 		else if (muonID == "embedding") return MuonID::EMBEDDING;
 		else if (muonID == "mediumhipsafe2016") return MuonID::MEDIUMHIPSAFE2016;
-		else return MuonID::NONE;
```

Comment:
Damn. I dont wanted to push this already :D
Anyway, this also introduces a check if a valid MuonID was given in the config.

Reply:
Looks good